admin/sql/updates/20200326-mbs-7781.sql

```diff
@@ -0,0 +1,79 @@
+\set ON_ERROR_STOP 1
+
+BEGIN;
+
+CREATE OR REPLACE VIEW duplicate_artist_credits AS
+    WITH ac_names AS (
+        SELECT ac.id,
+               array_agg(
+                row(acn.position,
+                    acn.artist,
+                    acn.name,
+                    acn.join_phrase)
+                ORDER BY acn.position ASC
+               ) AS names
+          FROM artist_credit ac
+          JOIN artist_credit_name acn
+            ON acn.artist_credit = ac.id
+         GROUP BY ac.id
+    )
+    -- The smallest ID will be the first in the array, which we'll prefer
+    -- as the "original" AC.
+    SELECT array_agg(id ORDER BY id ASC) AS duplicates
+      FROM ac_names
+     GROUP BY names
+    HAVING count(id) > 1;
+
+-- N.B. Affected entities and artist credits must be cleared from the cache
+-- after running this.
+
+CREATE OR REPLACE FUNCTION merge_duplicate_artist_credits(orig_ac INT)
+RETURNS void AS $$
+DECLARE
+    dupe_acs INT[];
+BEGIN
+    SELECT duplicates[2:]
```

Comment:
Test passed on CircleCI (job [1656](https://app.circleci.com/pipelines/github/mwiencek/musicbrainz-server/465/workflows/a46b2d49-1847-41ef-90b8-10432c777c82/jobs/1656)), not sure what is causing this failure on my host:
```
psql:t/pgtap/../../admin/sql/updates/20200326-mbs-7781.sql:77: ERROR:  syntax error at or near "]"
LINE 6:     SELECT duplicates[2:]
                                ^
t/pgtap/merge_duplicate_artist_credits.sql ............ Dubious, test returned 3 (wstat 768, 0x300)
No subtests run 
```

Reply:
Oh. I guess that's a feature in PG 9.6 (of course...). I don't see it mentioned in the 9.5 docs.

Added `array_length(duplicates, 1)` for the missing upper bound.